packages/document/docs/en/guide/start/cli.mdx

```diff
@@ -49,6 +49,25 @@ rsdoctor analyze --profile <manifestFile>
 rsdoctor analyze --profile "./dist/.rsdoctor/manifest.json"
 ```
 
+### `bundle-diff` Command
+
+The `bundle-diff` command is used to load **two** [manifest.json](/config/options/term#manifestjson) files **locally** and open the Rsdoctor [Bundle Diff](../usage/bundle-diff) page for **comparison and analysis of build bundles**.
```

Comment:
What would be great if possible is to generate the diff as a comment that we can automatically post to PRs

A bit similar to this: https://github.com/facebook/docusaurus/pull/10402#issuecomment-2289272099

Reply:
Ah, just read the blog post!

> Plans to support Bundle Diff Action on GitHub to simplify degradation monitoring during the CI phase.

Reply:
@slorber Thank you for the suggestion!  We will implement this feature.